module/Applications/view/applications/manage/detail.pdf.phtml

```diff
@@ -102,6 +102,7 @@ $this->headTitle($this->translate('Details'));
     <sethtmlpagefooter name="footer" value="on"/>
 <?php endif ?>
 <h1><?= $this->translate('application details') ?></h1>
+<p><?=sprintf($this->translate('You can see the application online on the %sfollowing link%s.'), '<a href="' . $this->serverUrl($this->url('lang/applications/detail', ['id' => $application->getId()])) . '">', '</a>')?></p>
```

Comment:
There is the possibility, that the user who applied is a "guest" (anonymous user), in which case the token must be included in the URL
Please have a look at [applications/apply/index.phtml 63-83](https://github.com/cross-solution/YAWIK/blob/9561de64106945f14d928a874a212b362f2e2d29/module/Applications/view/applications/apply/index.phtml#L63-L83)